packages/contracts/contracts/L2/teleportr/TeleportrDisbursement.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.9;
+
+contract TeleportrDisburser {
+    address private owner;
+    uint256 public totalDisbursements;
+
+    constructor() {
+        owner = msg.sender;
```

Comment:
We probably don't want the actual owner to be the deployer key. If we add a means of transferring ownership, then this might be OK. 

Same comment goes for the other constructor.

Reply:
Sounds good, I will check out how our other contracts do this today